app/models/spree/payment.rb

```diff
@@ -181,15 +181,25 @@ def revoke_adjustment_eligibility
       )
     end
 
+    # rubocop:disable Metrics/AbcSize, Metrics/LineLength
```

Comment:
It's better to refactor than introducing these lines. Or, if it's really out of scope, then add this file to the rubocop_todo list instead.

Reply:
I'll try and refactore that then 👍